utilities/workers/worker_manager.py

```diff
@@ -11,23 +17,105 @@ class WorkerManager:
-        """
-        Constructor creates internal queue and semaphore.
-        """
-        self.__workers = [] if workers is None else workers
-
-    def create_workers(self, count: int, target: "(...) -> object", args: tuple) -> None:  # type: ignore
+    @classmethod
+    def create(
+        cls,
+        count: int,
+        target: "(...) -> object",  # type: ignore
+        class_args: tuple,
+        input_queues: "list[queue_proxy_wrapper.QueueProxyWrapper]",
+        output_queues: "list[queue_proxy_wrapper.QueueProxyWrapper]",
+        controller: worker_controller.WorkerController,
+    ) -> "list[mp.Process]":
         """
-        Create identical workers.
+        Create identical workers and append them to a workers list.
 
         count: Number of workers.
         target: Function.
-        args: Arguments to function.
+        class_args: Arguments to function.
+        input_queues: Input queues.
+        output_queues: Output queues.
+        controller: Worker controller.
+
+        Returns whether the workers were able to be created and the Worker Manager.
         """
+
+        create_logger_result, worker_manager_logger = logger.Logger.create("worker_manager")
+        if not create_logger_result:
+            print("Error creating worker_manager_logger")
+            return False, None
+
+        frame = inspect.currentframe()
+        worker_manager_logger.info("worker manager logger initialized", frame)
+
+        if count <= 0:
+            return False, None
+
+        args = WorkerManager.create_worker_arguments(
+            class_args, input_queues, output_queues, controller
+        )
+
+        workers = []
         for _ in range(0, count):
+            result, worker = WorkerManager.create_single_worker(target, args, worker_manager_logger)
+            if not result:
+                frame = inspect.currentframe()
+                worker_manager_logger.error("Failed to create worker", frame)
+                return False, None
+
+            workers.append(worker)
+
+        return True, WorkerManager(
+            workers,
+        )
+
+    @staticmethod
+    def create_worker_arguments(
+        class_args: tuple,
+        input_queues: "list[queue_proxy_wrapper.QueueProxyWrapper]",
+        output_queues: "list[queue_proxy_wrapper.QueueProxyWrapper]",
+        controller: worker_controller.WorkerController,
+    ) -> tuple:
+        """
+        Creates a tuple containing most arguments for a worker.
+
+        class_args: Class arguments.
+        input_queues: Input queues.
+        output_queues: Output queues.
+        controller: Worker controller.
+
+        Returns a tuple with the arguments.
+        """
+        return class_args + tuple(input_queues) + tuple(output_queues) + (controller,)
+
+    @staticmethod
+    def create_single_worker(target: "(...) -> object", args: tuple, worker_manager_logger: logger) -> Tuple[bool, mp.Process]: #type: ignore
```

Comment:
Had to pass in the logger into this function because __init__ has not been called yet at this point in time. Will make it a class variable when needed.